### DISCUSSION TITLE
Dropdown not visible on Safari

### DISCUSSION THREAD

**Discussion by cdt-eth** [tailwindlabs/headlessui#1166] [Help]
I am using [the Simple Dropdown from TailwindUI](https://tailwindui.com/components/application-ui/elements/dropdowns#component-f8a14da22f26a67757b19f2fe3ca00ed) and it doesn't work on Safari. It doesn't expand or show any of the options, yet I do see them in `Inspect Mode`

Any thoughts?

<img width="509" alt="Screen Shot 2022-02-28 at 12 02 05 AM" src="https://user-images.githubusercontent.com/26611339/155926874-6edf972a-bf19-4c51-baa1-7bb9c7f2476b.png">
<img width="680" alt="Screen Shot 2022-02-28 at 12 05 21 AM" src="https://user-images.githubusercontent.com/26611339/155927115-1d47aa5c-38b0-491f-a3c3-160a921a9b95.png">

**Comment by RobinMalfait** [comment]
Looks like you are using an `option` tag. This doesn't work since it is a native element for the `<select>`. Use a div instead 👍

**Comment by cdt-eth** [comment]
thank you @RobinMalfait!
